nebula-logger/main/log-management/classes/LogEntryEventHandler.cls

```diff
@@ -178,28 +183,17 @@ public without sharing class LogEntryEventHandler {
         for (LogEntry__c logEntry : logEntryToTopics.keySet()) {
             for (String topicName : logEntryToTopics.get(logEntry)) {
                 // Add log entry-specific topics
-                topicAssignments.add(
-                    new TopicAssignment(EntityId = logEntry.Id, TopicId = topicNameToTopics.get(topicName).Id)
-                );
-
-                if (!LoggerSettings__c.getInstance().AddLogEntryTopicsToLog__c) {
-                    continue;
-                }
+                topicAssignments.add(new TopicAssignment(EntityId = logEntry.Id, TopicId = topicNameToTopics.get(topicName).Id));
 
                 // Add all topics to the parent log when enabled
-                topicAssignments.add(
-                    new TopicAssignment(EntityId = LOG.Id, TopicId = topicNameToTopics.get(topicName).Id)
-                );
+                topicAssignments.add(new TopicAssignment(EntityId = LOG.Id, TopicId = topicNameToTopics.get(topicName).Id));
             }
         }
         insert new List<TopicAssignment>(topicAssignments);
     }
 
     private void setOrganizationDetails() {
-        Organization org = [
-            SELECT Id, Name, InstanceName, IsSandbox, NamespacePrefix, OrganizationType
-            FROM Organization
-        ];
+        Organization org = [SELECT Id, Name, InstanceName, IsSandbox, NamespacePrefix, OrganizationType FROM Organization];
```

Comment:
I would definitely make this a private static final variable. I know it's being referenced in the constructor, but since we don't know how/if Salesforce boxcars platform event pushes, it's possible that multiple instances of the class could be instantiated from within the same context. The user query being re-done each possible time makes sense since that result might change.

Reply:
Yeah, fair enough, there's no good reason to do it within the loop - a private static variable makes more sense, I'll update it.